package.json

```diff
@@ -12,7 +12,8 @@
     "lint:eslint": "eslint '**/*.{js,jsx,ts,tsx}' --cache --format stylish",
     "fix": "yarn fix:eslint && yarn fix:prettier",
     "fix:prettier": "prettier --write '**/*.{js,jsx,ts,tsx}'",
-    "fix:eslint": "eslint '**/*.{js,jsx,ts,tsx}' --format stylish --fix"
+    "fix:eslint": "eslint '**/*.{js,jsx,ts,tsx}' --format stylish --fix",
+    "ts-node": "ts-node --compiler-options \"{\\\"module\\\":\\\"commonjs\\\"}\""
```

Comment:
The compiler options were added to make the `prisma db seed` command work with Next.js as mentioned in the [prisma docs](https://www.prisma.io/docs/guides/database/seed-database#compiler-options)